check-eve.py

```diff
@@ -77,6 +84,7 @@ def main():
     parser.add_argument("file", nargs="?", default=[])
     parser.add_argument("-q", dest="quiet", action="store_true")
     parser.add_argument("-s", dest="schema", action="store", required=True)
+    parser.add_argument("--min-seven", dest="minseven", action="store_true")
```

Comment:
I'd rather see this as `--check-empty-array` or something like that.

Reply:
Ok.
I thought it would fit for other additional checks...